UI/TechTreeWnd.cpp

```diff
@@ -41,6 +44,19 @@ namespace {
         db.Add("UI.tech-layout-zoom-scale",     UserStringNop("OPTIONS_DB_UI_TECH_LAYOUT_ZOOM_SCALE"),   1.0,  RangedStepValidator<double>(1.0, -25.0, 10.0));
         db.Add("UI.tech-controls-graphic-size", UserStringNop("OPTIONS_DB_UI_TECH_CTRL_ICON_SIZE"),      3.0,  RangedStepValidator<double>(0.25, 0.5,  12.0));
         db.Add("UI.windows." + RES_PEDIA_WND_NAME + ".persistently-hidden", UserStringNop("OPTIONS_DB_RESEARCH_PEDIA_HIDDEN"), false, Validator<bool>());
+
+        // TechListBox::TechRow column widths
+        int default_pts = 16;
+        db.Add(TECHROW_COL_XML_PREFIX + ".graphic",         UserStringNop(TECHROW_COL_ST_PREFIX + "_GRAPHIC"),
```

Comment:
not sure, but I don't think @adrianbroher would like this... Probably better to put the whole stringtable key in explicitly each time.